src/org/usfirst/frc/team4915/stronghold/commands/AutoCommand1.java

```diff
@@ -296,9 +215,9 @@ public static int getSpeed(Autonomous.Type type) {
         return speed;
     }
 
-    // Make this return true when this Command no longer needs to run execute()
     protected boolean isFinished() {
-        return false;
+        return super.isFinished(); // returns true if all the commands in this
```

Comment:
Er, never mind. thought this was in a `Command` class, not a `CommandGroup`. maybe just remove the entire method so it automatically runs `super.isFinished()`?